axon-server/administration/reset-event-processor-token.md

```diff
@@ -0,0 +1,146 @@
+Instances of `StreamingEventProcessor` can be obtained by querying the `eventProcessingConfiguration` method of your global configuration.
+```java
+configuration.eventProcessingConfiguration()
+    .eventProcessorByProcessingGroup(processorName,
+        StreamingEventProcessor.class)
+    ...
+```
+Calling `shutDown`, `resetTokens` and `start` on the retrieved `StreamingEventProcessor` performs the required steps to reset its token.
+This can be achieved as shown in the following code snippet:
+```java
+configuration.eventProcessingConfiguration()
+    .eventProcessorByProcessingGroup(processorName,
+        StreamingEventProcessor.class)
+    .ifPresent(streamingEventProcessor -> {
+        if (streamingEventProcessor.supportsReset()) {
+            streamingEventProcessor.shutDown();
+            streamingEventProcessor.resetTokens();
+            streamingEventProcessor.start();
+        }
+    });
+```
+
+Note that this only concerns stopping the local instances. 
+If there are instances running on other nodes, you either need to use AxonServer or build a solution for this yourself.
+For more details, refer to the section in the Axon Framework [reference guide](../../axon-framework/events/event-processors/streaming.md#triggering-a-reset).
+
+## Using the REST API
+Axon Server exposes a REST API to pause and start EventProcessors.
+When used in conjunction with the `resetTokens` method from earlier, this can be used to make sure that all instances of an EventProcessor are paused before a token is reset.
+The required parameters to do this are the following:
+- *component*: name of the component the EventProcessor is part of
+- *processor*: name of the EventProcessor itself
+- *context*: name of the context for which to reset the tokens
+- *tokenStoreId*: the identifier used to distinguish the desired token from other tokens stored in the same store. 
+
+Now 
+`/v1/components/{component}/processors/{processor}/pause?context={context}&tokenStoreIdentifier={tokenStoreId}`
+can be called, causing AxonServer to request all matched EventProcessors to stop.
+
+Then a reset as shown in the example using the framework can be performed.
+This is not Axon Server specific and hence is the same for all shown methods.
+
+As a final step, the EventProcessors can be started again with a patch request to the following url:
+`/v1/components/{component}/processors/{processor}/start?context={context}&tokenStoreIdentifier={tokenStoreId}` .
+
+
+## Using the Axon Server Connector
+To reset EventProcessors with the Axon Server Connector, the dependency has to be available on your classpath, 
+e.g. by using the following maven dependency.
+```xml
+<dependency>
+    <groupId>io.axoniq</groupId>
+    <artifactId>axonserver-connector-java</artifactId>
+    <version>4.6.0-SNAPSHOT</version>
```

Comment:
As this commit points to `master`, this will become the 4.6 documentation page. That means you can be certain there's a `4.6.0` release.
```suggestion
    <version>4.6.0</version>
```